deploy/distribute.py

```diff
@@ -47,37 +48,51 @@ def main(**kwargs):
     distributor_tx = kwargs['distributor_tx']
     auction_address = kwargs['auction']
     auction_tx = kwargs['auction_tx']
-    claims = kwargs['claims']
+    batch_number = kwargs['batch_number']
+
+    if batch_number:
+        batch_number = int(batch_number)
 
     with project.get_chain(chain_name) as chain:
         web3 = chain.web3
-        print("Web3 provider is", web3.currentProvider)
-        
```

Comment:
whitespace on a blank line! (there're other style warnings reported by flake8 in this file)

Reply:
running flake8 now